### PR TITLE
[MM-32896] Make the path configurable

### PR DESCRIPTION
**Pull request by ethervoid**
#### Summary

We need to make it possible for all the scripts to define their own path to pass the chart values

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-32896

### CODE REVIEW

**Review by stafot — APPROVED** [review]
Agree, looks as a reasonable approach to me. But please don't merge until @stylianosrigas also approves, because I might missing something.

**Review by stylianosrigas — APPROVED** [review]

**Pull request merged**
